src/shared/lib/ILazyMobXTableApplicationDataStore.ts

```diff
@@ -29,6 +33,8 @@ export class SimpleGetterLazyMobXTableApplicationDataStore<T> implements ILazyMo
     @observable public filterString:string;
     @observable public sortMetric:SortMetric<T>|undefined;
     @observable public sortAscending:boolean|undefined;
+    @observable public page:number;
+    @observable public itemsPerPage:number;
```

Comment:
disagree with @onursumer  - the goal of Simple... is to provide the a simple implementation of ILazyMobXTableApplicationDataStore. since the interface changed, Simple.. must change as well